ci/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-py38_4.10.3-Linu
     && conda init
 
 # install cuML
-ARG CUML_VER=23.06
+ARG CUML_VER=23.08
 RUN conda install -y -c conda-forge mamba=1.4.9 libarchive && \
-    mamba install -y -c rapidsai -c nvidia -c conda-forge cuml=$CUML_VER python=3.9 cuda-toolkit=11.5 \
+    mamba install -y -c rapidsai-nightly -c nvidia -c conda-forge cuml=$CUML_VER python=3.9 cuda-toolkit=11.5 \
```

Comment:
nit: rapids now use `cuda-version` pkg instead of cuda-toolkit to navigate the specific libcudf pkg
https://github.com/rapidsai/cudf#conda

Reply:
Thanks for the comment! Cuml is still using cuda-toolkit in the release selector: https://docs.rapids.ai/install. Maybe we can stay with it for now and try to use cuda-version after cuml adopts it.

Reply:
I think their page is out of date at least for 23.08 release https://github.com/rapidsai/cudf/pull/13615
but feel free to update it when needed